src/test/java/com/google/api/codegen/testdata/go_library.baseline

```diff
@@ -810,43 +794,40 @@ func ExampleClient_ListStrings() {
     _ = resp
 }
 
-func ExampleClient_AddComments() {
+func ExampleLibraryClient_AddComments() {
     ctx := context.Background()
-    c, err := library.NewClient(ctx)
+    c, err := library.NewLibraryClient(ctx)
     _ = err // Handle error.
 
     req := &google_example_library_v1.AddCommentsRequest{ /* Data... */ }
     err = c.AddComments(ctx, req)
     _ = err // Handle error.
 }
-============== file: google.golang.org/cloud/library/apiv1/library/doc.go ==============
+============== file: google.golang.org/cloud/library/apiv1/doc.go ==============
 // Copyright 2016 Google Inc. All Rights Reserved.
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+//      http://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
-// EDITING INSTRUCTIONS
-// This file was auto-generated and updates to that file get reflected
-// here through a refresh process. For the short term, the refresh
-// process will only be runnable by Google engineers. Manual additions are
-// allowed because the refresh process performs a 3-way merge in order to
-// preserve those manual additions. In order to notbreak the refresh
-// process, only certain types of modifications are allowed.
-//
-// Happy editing!
 
-// Package library provides the more language idiomatic and
-// developer friendly layer for the gRPC based API service.
-//
```

Comment:
What happened to the comment we came up with yesterday?

`Package library is an experimental, auto-generated ....`